src/metaObject.cxx

```diff
@@ -24,6 +24,9 @@
 namespace METAIO_NAMESPACE {
 #endif
 
+// Do not enforce c++11 requirement here, prefer storing the result of
+// std::numeric_limits<double>::max_digits10:
+#define MAX_DIGITS10 17
```

Comment:
In case dependent headers already define this (unlikely but why not), prefer `METAIO_MAX_DIGITS10`